docs/open-source-tizen/tizen-studio/emulator-extension-guide.md

```diff
@@ -1,142 +1,138 @@
+    - `args4j`, `jfxrt`, `jline`, `json-simple`, and `protobuf`
+3. Build the code:
+   - In the case of `lib`, `ui`, and `cli` libraries, to build them all at once, enter the `ant` shell command at the top level of your project.
+   - In the case of devices:
+     - When building is complete, copy the built `libecp`, `emulator-control-panel.jar`, and `emulator-control-panel-cli.jar` files to `libs` in the devices project directory.
+     - To build the `mobile`, `wearable`, and `tv` profiles all at once, enter the `ant` shell command at the top level of your project. To build a specific profile only, you can specify, for example, `ant -f ECP-DEVICE/build_mobile.xml`.
+4. If you configure devices separately, you only need to push changes to the `emulator-control-panel-devices` repository.
+5. When building is complete, move the following files:
+    - `libecp.jar`, `emulator-control-panel.jar`, and `emulator-control-panel-cli.jar` to `sdk/tools/emulator/bin`
+    - Device `xml` and `jar` files to `platforms/<version>/<profile>/emulator-resources/plugins`
+        - You must also rename the `.jar` and `.xml` files as 'ecp-plugin-<profile>-<version>.<extension>'.
+
+To execute the extension in Eclipse:
+
+1. Launch the Eclipse IDE.
+2. Import all the project files.
+3. In the run configuration, set the main project and class.
+   - Project: `ECP-UI` for UI or `ECP-CLI` for CLI
+   - Main class: `org.tizen.ecp.EmulatorControlPanel` for UI or `org.tizen.cli.TizenEmulatorCli` for CLI
+4. In the run configuration, set the program arguments.
+   - vm_name: the name of vm instance
+   - base_port: this can be found the first item from the context menu when you right click on an emulator. The port indicates the connected sdb port, so you should subtract 1 to get the base port.
+   - platform_version: see below example
+   - profile: see below example
+   For example:
+   ```
+   vm_name=w-0906-1 base_port=26100 platform_version=tizen-2.3.2 profile=wearable
+   ```
+   The platform version and profile are used in the directory names:
+   ```
+   <tizen-sdk>/platforms/<platform_version>/<profile>
+   ```
+5. In the run configuration, add the `ECP-DEVICES` project into class path and source path.
+6. In the run configuration, enter the following VM arguments. This is needed in order to change the search option to find the device xml and jar location. Basically, non-develop mode of ECP searches 3 above relative directory to find sdk.info. If it is develop mode, it searches current directory and the executable binary directory.
+    - `-Ddevelop`
+7. Create a symbolic link on a terminal to set target directory of tizen studio. Tizen studio base location is used to find the target device xml and jar location, including extension version and profile location.
+    - `sdk.info` file is found based on the SDK installation path
+    - For example: `emulator-control-panel/ECP-LIB`
+      - `ln -s ~/tizen-studio/sdk.info sdk.info`
+
+### Changes
```

Comment:
It is still unclear what the purpose of this section is.

Exactly what kind of "changes" are these?

Are they changes the developer must make while developing an emulator extension?
Or are the changes compared to how the extensions were developed before?
Or maybe something else entirely?

Can you please explain this? Basically, we need an intro sentence in this section before the bullet list that states what these changes are for and when the developer needs to pay attention to them.

Reply:
It has been a long time since it was changed, so this section may be omitted.